app/helpers/review_count_helper.rb

```diff
@@ -1,5 +1,5 @@
 module ReviewCountHelper
-  def number_of_cards_for_review(show_text: false, text: "Осталось вспомнить ")
+  def number_of_cards_for_review(show_text: false, text: t('cards.for_review'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.